src/contexts/ClientContext.tsx

```diff
@@ -49,5 +49,11 @@ export function ClientProvider({ children }: React.HTMLAttributes<HTMLOrSVGEleme
     setLoaded(true);
   }, [ibcClient, tmClient]);
 
-  return loaded ? <ClientContext.Provider value={value}>{children}</ClientContext.Provider> : <></>;
+  return loaded ? (
```

Comment:
Can you rename `loaded` to `clientsAvailable` or something like that? Setting up `tmClient`/`ibcClient` happens before the actual request. There is indeed a network request involve, but this is not necessarily the case and can change over time to a purely local setup.